### PR TITLE
fix links in Readme.md

### LINKED ISSUE #751: Links in ReadMe are broken
**Description**
Some links in readme are not up-to-date

### PR DESCRIPTION
**Pull request by PolyDevil**
# Pull Request

## What does this PR do?
Fixes #751

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ NB: If you don't have any Meilisearch instance running and containing your data,
 - [🔧 Installation](#-installation)
 - [🎬 Usage](#-usage)
 - [💅 Customization](#-customization)
-- [⚡️ Example with InstantSearch](#-example-with-instantSearch)
+- [⚡️ Example with InstantSearch](#-example-with-instantsearch)
```
Comment (bidoubiwa) [inline README.md:44]: This one still does not work :/  These seems to be a weird behavior when I look at the anchor. 

> %EF%B8%8F-example-with-instantsearch

Could you fix it?

Comment (PolyDevil) [reply README.md:44]: I will look into that, sure.
The behavior is weird, so probably will look into how github handles the anchor links and emoji.

Comment (bidoubiwa) [reply README.md:44]: Feel free to change the emoji if it is too annoying. You can chose 🔥

**Comment by bidoubiwa** [comment]
I'm merging this into an appart branch to apply the missing changes! 
Thanks again for spotting this @PolyDevil

**Pull request merged**